src/submission/tests.py

```diff
@@ -566,7 +566,7 @@ def test_article_encoding_bibtex(self):
                 title = {Test article: a test article},
                 volume = {2},
                 year = {1990},
-                url = {http://localhost/TST/article/id/%s/},
+                url = {http://testserver/article/id/%s/},
```

Comment:
For this change to work consistently, the test needs to run under domain mode explicitely, otherwise the test outcome can change based on user settings. See this for an example:

https://github.com/BirkbeckCTP/janeway/blob/5511797544a27e30751b156474b08da42f401fb6/src/repository/tests/test_views.py#L58